doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -260,6 +260,17 @@ Pin to numeric bookmark               :kbd:`1` … :kbd:`9`
 ===================================== =============================================
 ```
 
+### Crop window tool ###
```

Comment:
Can we put this after the Transform Tools section? The CropWindow tool is secondary to the TransformTools, and this section isn't all that information-rich anyway.

Reply:
https://github.com/GafferHQ/gaffer/pull/3582/commits/32e6da55aea89ea2ac9d76936d7ea0e9532ecd97 moved to end.